src/test/java/com/zufar/onlinestore/product/endpoint/e2e/ProductsEndpointTest.java

```diff
@@ -0,0 +1,80 @@
+package com.zufar.onlinestore.product.endpoint.e2e;
```

Comment:
I think we can delete e2e package and just leave `com.zufar.onlinestore.product.endpoint` package